packages/resource-deployment/scripts/deploy-rest-api.sh

```diff
@@ -47,23 +47,25 @@ fi
 
 echo "Starting deployment for REST api..."
 
-echo "Deploying named values"
-deployResource "$apiTemplates/model-namedValues.template.json"
+# Commenting out empty template files, since each action adds 30-40 secs of deployment time.
+# We may need to have a single ARM template to improve performance.
+# echo "Deploying named values"
```

Comment:
may be creat bug instead of putting comment here?

Reply:
created issue https://github.com/microsoft/accessibility-insights-service/issues/611